qiskit_ibm_runtime/accounts/account.py

```diff
@@ -30,6 +30,7 @@
 
 IBM_QUANTUM_API_URL = "https://auth.quantum-computing.ibm.com/api"
 IBM_CLOUD_API_URL = "https://cloud.ibm.com"
+PRIVATE_EU_API_URL = "https://private.eu-de.quantum-computing.cloud.ibm.com/"
```

Comment:
Not sure if it is a good idea to have a constant per region. I think it would work better the current approach, where you build the url depending on the region coming in the CRN. After that, if the `private_endpoint` param is provided add `private` subdomain at the very beginning of the url. In summary, the pattern is `https://<private>.<region>.<subdomain>.cloud.ibm.com` where:
   - `<private>` is only added if `private_endpoint` is `True`.
   - `<region>` is extracted from the CRN.